### DISCUSSION TITLE
Which blood group is sometimes called the "universal donor"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#372] [Galaxy Trivia] [answered]
This is because it has the lowest risk of causing serious reactions for most people who receive it.

**Comment by thuongtruong1009** [comment, marked as answer]
Blood group O is sometimes called the "universal donor" because people with this blood type can donate blood to individuals with any other blood type without causing an adverse immune reaction. This is because blood group O does not have any A or B antigens on the surface of its red blood cells, which can trigger an immune response in individuals with blood types A, B, or AB. However, it's important to note that while blood group O is considered a "universal donor" for red blood cells, it may not be the best match for other components of blood such as plasma and platelets.

**Reply by ParanoidUser** [reply]
Well done, could you specify RhD?

**Comment by peterbaccaro** [comment]
O negative is sometimes called the universal blood donor type.
